docs/use/createBackups/EnablePITR.md

```diff
@@ -0,0 +1,44 @@
+# Point-in-time recovery (PITR) backups
+
+PITR maintains versions of your database from past timestamps, serving as a safeguard against data loss during various disasters, including database crashes, accidental data deletions, table drops, or unintended updates to multiple fields instead of a single one.
+
+PITR complements on-demand and scheduled backup strategies by providing finer backup granularity with more specific recovery points for restoring data to the same cluster.
+
+## How it works
+
+Restoring databases up to a specific moment in time involves retrieving data from a backup snapshot and replaying all subsequent events that occurred until a specified moment using oplog slices.
+
+When PITR and backups are enabled (either on-demand or scheduled), Everest starts capturing successive oplog slices at intervals. As soon as the initial full backup is available, Everest can start restoring the database from these PITR oplogs.
+
+Since Everest saves oplog slices and streams them into your storage between scheduled task runs, scheduling frequent PITR backups is not necessary. You can use the available oplog slices in your storage to restore a backup to any moment between snapshots.
```

Comment:
```suggestion
Since Everest saves transaction logs and streams them into your storage between scheduled task runs, scheduling frequent backups is not necessary. You can use the available transaction logs in your storage to restore a backup to any moment between snapshots.
```